game.py

```diff
@@ -1,19 +1,25 @@
 from board import Board
 import copy
+import random
 
 class Game:
-	def __init__(self, board_size, board, start_player=0):
+	def __init__(self, board_size, board, player=0):
 		self.board_size = board_size
 		self.board = board
-		self.current_player = start_player
+		self.current_player = player
 		self.player_symbol = ('x','o')
 	def play(self):
 		pass
 	# Returns a boolean of whether the game is over
 	# We need the inputs because we will want it to differ from self when we are down in the minimax tree
 	# We might not need self? 
 	def end_state(self, board, current_player):
-		return False
+		if empty(self.get_legal_moves(current_player)):	# i.e. if the current_player has no more moves then either: the current_player has no pieces left or has no legal moves
+			winning_player = 1 + self.current_player
```

Comment:
If current player is 1, this goes to 2.`1-self.current_player` switches 1->0 and 0->1